Makefile

```diff
@@ -5,7 +5,7 @@ SHELL = /bin/sh
 MAKEFLAGS += -j2
 
 export DOCKER_IMAGE_NAME ?= osparc-map
-export DOCKER_IMAGE_TAG ?= 0.0.9
```

Comment:
Out of curiosity: Is this version/tag set in a single place and then automatically updated in all locations? I can see it appears in several places so if that's not the case then I would suggest to add such a mechanism (e.g. by picking it from an env var or having a `VERSION` file where it is picked from.

Reply:
no, this is all done using the 'make version-patch' command. I didn't invent this, I think i took the plain Makefile from another dynamic service (probably jupyter-math or so).